### PR TITLE
[986] react modules UI

### PR DESCRIPTION
**Pull request by jremen**
**Changes proposed in this Pull Request**
Preliminary react modules UI

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/wp-urlslab#986

### CODE REVIEW

File: admin/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  const [count, setCount] = useState(0);
+
+  const testingFilterItems = {
+    0: 'Most reviews from external portals',
+    1: 'Best ratings from external portals',
+    2: 'Out best rating',
+    3: 'Recently updated'
+  };
+
+  // const createPost = fetch('http://liveagent.local/wp-json/wp/v2/posts', {
```
Comment (vzeman) [inline admin/src/App.jsx:24]: remove such comments

File: admin/src/components/DashboardModule.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		}
+		);
+	}
+
+	return (
+		<div className={`urlslab-dashboardmodule ${activating ? 'activating' : ''} ${moduleActive ? 'active' : ''}`}>
+			{hasApi
+				? <div className="urlslab-dashboardmodule-api">
+					<ApiIcon />
+					This module requires API key
```
Comment (vzeman) [inline admin/src/components/DashboardModule.jsx:28]: change text to shorter version with option to translate: __('Urlslab API key required')

File: admin/src/constants/variables.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+		return '/app/plugins/urlslab-plugin/admin/public'
+	}
+	return '';
+}
+
+export const domain = () => {
+	if ( production ) {
+		return ''
+	}
+	return 'http://liveagent.local';
```
Comment (vzeman) [inline admin/src/constants/variables.js:14]: never like this pls ... domain should be computed from browser window url

File: admin/src/data/modules.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
+[
```
Comment (vzeman) [inline admin/src/data/modules.json:1]: do we need this file? it looks like test file before we had api ready
pls remove

**Review by vzeman — CHANGES_REQUESTED** [review]

File: admin/src/App.jsx (hunk 1 above)
Comment (jremen) [reply admin/src/App.jsx:24]: Not used module, only to demo UI elements – will be removed later

File: admin/src/constants/variables.js (hunk 3 above)
Comment (jremen) [reply admin/src/constants/variables.js:23]: removed entirely – used for running in custom dev server outside of WP

File: admin/src/data/modules.json (hunk 4 above)
Comment (jremen) [reply admin/src/data/modules.json:1]: not anymore

**Pull request closed without merging**